microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/integration/FailSafeCDICustomConfigSourceProviderHealthTest.java

```diff
@@ -26,12 +29,16 @@
 @RunWith(Arquillian.class)
 @RunAsClient
 @ServerSetup({ DisableDefaultHealthProceduresSetupTask.class, FailSafeCDICustomConfigSourceProviderHealthTest.SetupTask.class })
+@Category(HealthWithFaultToleranceTests.class)
 public class FailSafeCDICustomConfigSourceProviderHealthTest extends FailSafeCDIHealthDynamicBaseTest {
     private static final String PROPERTY_FILENAME = "health.properties";
     Path propertyFile = Paths.get(
-            FailSafeCDICustomConfigSourceProviderHealthTest.class.getResource(PROPERTY_FILENAME).getPath());
+            FailSafeCDICustomConfigSourceProviderHealthTest.class.getResource(PROPERTY_FILENAME).toURI());
     private byte[] bytes;
 
+    public FailSafeCDICustomConfigSourceProviderHealthTest() throws URISyntaxException {
```

Comment:
What is the reason for introducing the constructor?

Reply:
A field initialization has been fixed by adding `toURI()`, which can throw `URISyntaxException`, which must be either caught, or declared as part of the ctor signature.